tails_files/install.sh

```diff
@@ -61,14 +83,19 @@ if $ADMIN; then
   echo "# HidServAuth lines for SecureDrop's authenticated hidden services" | cat - $ANSIBLE/app-ssh-aths $ANSIBLE/mon-ssh-aths $ANSIBLE/app-document-aths > $ADDITIONS
   if [[ -d "$HOMEDIR/.ssh" && ! -f "$HOMEDIR/.ssh/config" ]]; then
     # create SSH host aliases and install them
-    SSHUSER=$(zenity --entry --title="Admin SSH user" --window-icon=$INSTALL_DIR/securedrop_icon.png --text="Enter your username on the App and Monitor server:")
+    # parse prod-specific.yml YAML into Bash variables
+    eval $(parse_yaml $ANSIBLE/prod-specific.yml)
+    if [[ -z $ssh_users ]]; then
+        echo "Error: ssh_users is not defined in prod-specific.yml." 1>&2
+        exit 1
+    fi
     cat > $INSTALL_DIR/ssh_config <<EOL
 Host app
   Hostname $APPSSH
-  User $SSHUSER
+  User $ssh_users
 Host mon
   Hostname $MONSSH
-  User $SSHUSER
+  User $ssh_users
```

Comment:
@conorsch I think we've been over this before, so sorry if I'm forgetting the conclusion. What does Ansible do if `ssh_users` is a list? Is that allowed, or does it really expect only one username to be listed there?

My memory is that it only expects one username to be listed there, and the only reason we haven't renamed that field to `ssh_user` is because it's in the version-controlled `prod-specific.yml`, and so change it will be disruptive to the typical stash-pull-pop update workflow for admins.